### DISCUSSION TITLE
Download oldest photos

### DISCUSSION THREAD

**Discussion by jonassonjp** [icloud-photos-downloader/icloud_photos_downloader#609] [General]
Hey guys, awesome python script! Many thanks!
Is there a way to download the oldest photos from iCloud library?

**Comment by AndreyNikiforov** [comment]
> Is there a way to download the oldest photos from iCloud library?

What exactly do you want to achieve? Running `icloudpd` will take care of all photos in your collection, old and new. If you already have new photos locally, then only old ones will be downloaded.

**Comment by jonassonjp** [comment]
I was thinking in downloading and deleting the old one first so I would organize them locally

**Reply by AndreyNikiforov** [reply]
> I was thinking in downloading and deleting the old one first so I would organize them locally

What is different in your scenario of downloading and processing images in chunks vs downloading all and processing in chunks?

**Comment by jonassonjp** [comment]
Because I want to download the old ones first, delete them from iCloud and
start to organize locally.
Do you know if there is a way?

best reggards

-jonas

**Reply by AndreyNikiforov** [reply]
> Because I want to download the old ones first, delete them from iCloud and start to organize locally. Do you know if there is a way? 

You can start `icloudpd` and then cancel download when it finishes old photos.

**Comment by jonassonjp** [comment]
Ok, thank you
[]´s

-jonas
